docs/docs/cmd/spo/list/list-get.md

```diff
@@ -29,18 +32,30 @@ m365 spo list get [options]
 
 ## Examples
 
-Return information about a list with ID _0cd891ef-afce-4e55-b836-fce03286cccf_ located in site _https://contoso.sharepoint.com/sites/project-x_
+Get information about a list with ID _0cd891ef-afce-4e55-b836-fce03286cccf_ located in site _https://contoso.sharepoint.com/sites/project-x_
 
 ```sh
 m365 spo list get --id 0cd891ef-afce-4e55-b836-fce03286cccf --webUrl https://contoso.sharepoint.com/sites/project-x
 ```
 
-Return information about a list with title _Documents_ located in site _https://contoso.sharepoint.com/sites/project-x_
+Get information about a list with title _Documents_ located in site _https://contoso.sharepoint.com/sites/project-x_
 
 ```sh
 m365 spo list get --title Documents --webUrl https://contoso.sharepoint.com/sites/project-x
 ```
 
+Get information about a list with server relative url _sites/project-x/Documents_ located in site _https://contoso.sharepoint.com/sites/project-x_
+
+```sh
+m365 spo list get --title Documents --webUrl https://contoso.sharepoint.com/sites/project-x --listUrl 'sites/project-x/Documents'
```

Comment:
I think the option we added it `url` not `listUrl`

```suggestion
m365 spo list get --title Documents --webUrl https://contoso.sharepoint.com/sites/project-x --url 'sites/project-x/Documents'
```

also in the example below

Reply:
also I just noticed it will not work as we specify `title` and `listUrl` (`url`) at the same time 😉